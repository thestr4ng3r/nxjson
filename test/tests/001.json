    {
      "some-int": 195,
      "array1": [ 3, 5.1, -7, "nine", /*11*/, true, false, "last" ],
      "array2":["/*","*/"],
      "some-bool": true,
      "other-bool": false,
      "some-dbl": -1e-4,
      "some-null": null,
      "hello": "world!",
      "str1": "//",
      "str2": "\\",
      "str3": "text /*text*/ text",
      "str4": "\\text\\",
      "str5": "\?text\?",
      "str\t6\\": "text\ntext\ttext",
      "str7": "text\u1234text\u5678",
      //"other": "/OTHER/",
      "obj": {"KEY":"VAL", "obj":{"KEY":"VAL"}}
    }
