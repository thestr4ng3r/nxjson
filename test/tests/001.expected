{
  some-int:195
  array1:[
    3
    5.100000e+00
    -7
    "nine"
    T
    F
    "last"
  ]
  array2:[
    "/*"
    "*/"
  ]
  some-bool:T
  other-bool:F
  some-dbl:-1.000000e-04
  some-null:null
  hello:"world!"
  str1:"//"
  str2:"\"
  str3:"text /*text*/ text"
  str4:"\text\"
  str5:"\?text\?"
  str	6\:"text
text	text"
  str7:"textሴtext噸"
  obj:{
    KEY:"VAL"
    obj:{
      KEY:"VAL"
    }
  }
}
